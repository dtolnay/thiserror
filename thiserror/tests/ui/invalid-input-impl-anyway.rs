use thiserror::Error;

#[derive(Error, Debug)]
#[error]
pub struct MyError;

fn main() {
    // No error on the following line. Thiserror emits an Error impl despite the
    // bad attribute.
    _ = &MyError as &dyn std::error::Error;
}
