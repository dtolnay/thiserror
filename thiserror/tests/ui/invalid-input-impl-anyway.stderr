error: expected attribute arguments in parentheses: #[error(...)]
 --> tests/ui/invalid-input-impl-anyway.rs:4:3
  |
4 | #[error]
  |   ^^^^^
